src/main/java/pl/lodz/zzpj/kanbanboard/service/UserService.java

```diff
@@ -19,12 +26,39 @@
 
     private final DateProvider dateProvider;
 
+    @Autowired
+    PasswordEncoder passwordEncoder;
+
     @Autowired
     public UserService(UsersRepository usersRepository, DateProvider dateProvider) {
         this.usersRepository = usersRepository;
         this.dateProvider = dateProvider;
     }
 
+    @PostConstruct
+    public void fillRepo() throws BadOperationException {
```

Comment:
It should not be placed inside a service.  I recommend a separate class (UserDataFiller?)